pkgs/development/python-modules/imageio/default.nix

```diff
@@ -3,23 +3,27 @@
 , fetchPypi
 , pillow
 , psutil
+, imageio-ffmpeg
 , pytest
 , numpy
 , isPy3k
+, ffmpeg
 , futures
 , enum34
 }:
 
 buildPythonPackage rec {
   pname = "imageio";
-  version = "2.4.1";
+  version = "2.5.0";
 
   src = fetchPypi {
-    sha256 = "0jjiwf6wjipmykh33prjh448qv8mpgngfi77ndc7mym5r1xhgf0n";
+    sha256 = "1bdcrr5190jvk0msw2lswj4pbdhrcggjpj8m6q2a2mrxzjnmmrj2";
     inherit pname version;
   };
 
-  checkInputs = [ pytest psutil ];
+  checkInputs = [ pytest psutil ] ++ stdenv.lib.optionals isPy3k [
+    imageio-ffmpeg ffmpeg
+  ];
   propagatedBuildInputs = [ numpy pillow ] ++ stdenv.lib.optionals (!isPy3k) [
     futures
     enum34
```

Comment:
I suspect this is where you could add `pathlib` (for https://github.com/imageio/imageio/issues/436)

Reply:
@almarklein Thanks, that's exactly where I put them.